src/views/Project/index.jsx

```diff
@@ -282,6 +282,12 @@ export default class Project extends Component {
     </a>
   );
 
+  handleButtonClick = () => {
+    this.setState(prevState => ({
+      introductionOpen: !prevState.introductionOpen,
+    }));
```

Comment:
Nothing wrong with yours but just want to keep the same flavour of `this.setState()` throughout the app.
```jsx
this.setState({
  introductionOpen: !this.state.introductionOpen,
});
```